course1/module2/Question2.js

```diff
@@ -0,0 +1,94 @@
+// Javascript Methods.
+
+//Declaring Array.
+let v = new Array(1,2,3,'A','B','C',"Harshal","Jayesh","Sonawane",null);
```

Comment:
You can declare an array without using the new Array keywords aswell.
eg let v = [1,2,3,'a','b']
This will also work fine